docs/development/search.rst

```diff
@@ -0,0 +1,99 @@
+Search
+============
+
+Read The Docs uses Elasticsearch_ instead of the built in Sphinx search for providing better search
+results. Documents are indexed in the Elasticsearch index and the search is made through the API.
+All the Search Code is open source and lives in the `Github Repository`_.
+Currently we are using the `Elasticsearch 6.3`_ version.
+
+Local Development Configuration
+-------------------------------
+
+Installing and running Elasticsearch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You need to install and run the `Elasticsearch 6.3`_ version on your local development machine.
```

Comment:
...run Elastichsearch version `6.3`_ on your... 

This might be a bit more fluent.